src/components/ColorSwatches/index.js

```diff
@@ -0,0 +1,144 @@
+  const sampleColumn4 = ['#818181', '#7f7e09', '#03807a', '#641c5c', '#a2a1a7'];
+
+  const colorsArray = [
+    sampleColumn1,
+    sampleColumn2,
+    sampleColumn3,
+    sampleColumn4,
+  ];
+
+  useEffect(() => {
+    const isSystemColor = checkIfSystemColor(currentColor, colorsArray);
+    if (isSystemColor) {
+      setCustomColor(false);
+    } else {
+      setCustomColor(currentColor);
+    }
+  }, [currentColor]);
+
+  const checkIfSystemColor = (colorToCheck, systemColors) => {
+    let existSystemColor = false;
+    for (const colorCode of systemColors) {
+      if (colorCode.includes(colorToCheck)) existSystemColor = true;
+    }
+    return existSystemColor;
+  };
+
+  const handleColorSelect = selectedColor => {
+    setDesktopState(prev => ({ ...prev, color: selectedColor }));
+    setOpenColorSwatches(false);
+  };
+
+  return (
+    <Swatches>
+      <section className="inline-grid">
+        {colorsArray.map(column => (
+          <div key={column[0]}>
+            {column.map(color => (
+              <SampleBox
+                onClick={() => handleColorSelect(color)}
+                key={color}
+                sampleColor={color}
+                highlighted={color === highlighted}
+                onMouseEnter={() => setHighlighted(color)}
+              />
+            ))}
+          </div>
+        ))}
+      </section>
+      <hr />
+      <section className="inline-grid">
+        <Button>
+          <input
+            type="color"
+            className="customColorPicker"
+            onChange={e => handleColorSelect(e.target.value)}
+          />
+          Other...
+        </Button>
+        <SampleBox
+          hidden={!customColor}
+          sampleColor={customColor}
+          highlighted={customColor === highlighted}
+          onMouseEnter={() => setHighlighted(customColor)}
+          onClick={() => handleColorSelect(customColor)}
+        />
+      </section>
+    </Swatches>
+  );
+}
+
+const Swatches = styled.div`
+  position: absolute;
+  width: 95px;
+  height: 153px;
+  padding-left: 1px;
+  background-color: rgb(236, 233, 218);
+  box-sizing: border-box;
+  outline: 1px outset white;
+  -webkit-box-shadow: 2px 2px 3px -1px rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 2px 2px 3px -1px rgba(0, 0, 0, 0.75);
+  box-shadow: 2px 2px 3px -1px rgba(0, 0, 0, 0.75);
+
+  & .customColorPicker {
+    position: absolute;
+    width: 61px !important;
+    height: 18px;
+    opacity: 0;
+    left: 4px;
+    bottom: 5px;
+  }
+
+  & .inline-grid {
+    display: flex;
+  }
+
+  & hr {
+    width: 88px;
+    margin: -1px 0px 2px 2px;
+  }
+  & button {
+    width: 63px !important;
+    margin: 1px 2px 0px 2px;
+  }
+`;
+
+const SampleBox = styled.div`
+  display: ${props => (props.hidden ? 'none' : 'block')}
+  width: 17px;
+  height: 17px;
+  background-color: ${props => props.sampleColor}
+  -webkit-box-shadow: inset 2px 2px 2px 0px rgba(0,0,0,0.5);
+  -moz-box-shadow: inset 2px 2px 2px 0px rgba(0,0,0,0.5);
```

Comment:
The only box shadow needed is `box-shadow` with no vendor prefixes, as this **[2012 article](https://css-tricks.com/do-we-need-box-shadow-prefixes/)** on css tricks says.